docs/creating_service_module.md

```diff
@@ -0,0 +1,30 @@
+## Creating a new Service Module project in this repository
+
+This section walks through adding a new Service Module project to the SDK repository.
+
+### Creating the project
+
+New Service Modules in this repository should be created under the `org.aerogear.mobile.*` namespace.
+
+See the [`keycloak-service-module`](../keycloak-service-module/build.gradle) for examples.
+
+1. Via Android Studio
+
+    1. Create a new Android Library model by navigating to `File > New > New Module...` and selecting the `Android Library` option.
+
+    2. Input the new module name and ensure it is created under the `org.aerogear.mobile.*` namespace. Ignore the value of the `Minimum SDK` field for now.
+
+    3. After the module is created, update the fields in the generated `build.gradle` file to target the corresponding constants in the [`constants.gradle`](../constants.gradle) file. 
+
+
+2. Manually
+
+Create a new folder in the root of the repository with the `/*-module` suffix, with a `build.gradle` and `gradle.properties` file describing it.
```

Comment:
This section (2. Manually) is missing a numbered list like there is in the first one (1.Via Android Studio).
1. ..
    1. Create a new folder...
    2. Target corresponding constants...
    3. Add module to settings.gradle